local_build.sh

```diff
@@ -0,0 +1,2 @@
+doit dist
+pip  install --force-reinstall dist/ruly-0.0.5-py3-none-any.whl
```

Comment:
Not sure if this is needed - if I understood correctly, you used it to have a "dynamically updating" lib implementation, so there's a faster feedback loop between changing stuff and using the library? If so, it's a use case already covered by `pip install -e .`.

Reply:
I was running into issues with ruly-dmn where it was pointing to the ruly-0.0.4 which is why I made sure to force to the fork version.